spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkSessionCatalog.java

```diff
@@ -86,4 +87,26 @@ public void testValidateHmsUri() {
             .defaultNamespace()[0]
             .equals("default"));
   }
+
+  @Test
+  public void testLoadFunction() {
+    String envHmsUriKey = "spark.hadoop." + METASTOREURIS.varname;
+    String catalogHmsUriKey = "spark.sql.catalog.spark_catalog.uri";
+    String hmsUri = hiveConf.get(METASTOREURIS.varname);
+
+    spark
```

Comment:
Now that this is in two places maybe it makes sense to put this in a "beforeAll" method. Also when we do that we do that and extract this from the above test as well, we can do

spark.conf().set("spark.sql.catalog.spark_catalog", SparkSessionCatalog.class.getName) 

If that fits on one line